.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
     "files.watcherExclude": {},
-    "WhiteSource Advise.Diff.BaseBranch": "main"
+    "WhiteSource Advise.Diff.BaseBranch": "main",
+    "eslint.workingDirectories": [

```

Comment:
This ensures to run eslint inside sub-packages only and this also fixes the issue with the different cwds